src/simgear/props/props.cxx

```diff
@@ -838,12 +838,12 @@ const int SGPropertyNode::LAST_USED_ATTRIBUTE = PRESERVE;
  * Default constructor: always creates a root node.
  */
 SGPropertyNode::SGPropertyNode ()
-  : _index(0),
-    _parent(0),
+  : _parent(0),
+    _listeners(0),
```

Comment:
```suggestion
  : _parent(nullptr),
    _listeners(nullptr),
```
I suggest while we are at it to use C++11's `nullptr` instead of the old-fashioned `NULL` or even worse `0`.